components/add_groups_to_channel_modal/__snapshots__/add_groups_to_channel_modal.test.tsx.snap

```diff
@@ -118,9 +118,7 @@ exports[`components/AddGroupsToChannelModal should match when renderOption is ca
     <div
       className="more-modal__name"
     >
-      <Nbsp />
-      -
-      <Nbsp />
+       - 
```

Comment:
@nevyangelova How do i fix this snapshot ? Can't use NBSP component as it's return type isn't valid JSX.Element and typescript complains ?

Reply:
@Ampit didn't the automatic command update the snaps correctly? `npm test -- -u`

Reply:
@nevyangelova Nope this is the updated snap ?

Reply:
have you tried resolving the conflict? I can't personally view it because I am not the author of the PR. But it should potentially take your changes and work

Reply:
![image](https://user-images.githubusercontent.com/2737004/107551493-56cac200-6ba0-11eb-81dc-dff27761da52.png)

It won't allow me to do it here and Everything seems to be fine on my end when i run the tests.

Reply:
Then i would suggest try updating your fork to our latest master and then update this branch and resolve the conflicts in your editor :)